CHANGES.rst

```diff
@@ -43,6 +43,9 @@ astropy.io.misc
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Fixed bug in ``ColDefs._init_from_array() that caused non-scalar unsigned
```

Comment:
Formatting typo.
```suggestion
- Fixed bug in ``ColDefs._init_from_array()`` that caused non-scalar unsigned
```

Reply:
Also need to move this entry to a section that is consistent with milestone. Thanks!